eth-bridge/contracts/src/Bridge.sol

```diff
@@ -6,31 +6,8 @@ import {UUPSUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/prox
 import {AccessControlUpgradeable} from
     "openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
 import {WrappedToken} from "./WrappedToken.sol";
-
-/// Struct for representing Substrate -> ETH transfer
-struct IncomingReceiptStruct {
```

Comment:
Extracted to `BridgeTypes.sol`, as new solhint version doesn't like multiple contracts in one file.